src/main/scala/com/microsoft/hyperspace/actions/OptimizeAction.scala

```diff
@@ -135,11 +135,13 @@ class OptimizeAction(
   override def logEntry: LogEntry = {
     // Update `previousIndexLogEntry` to keep `filesToIngore` files and append to it
     // the list of newly created index files.
-    val absolutePath = PathUtils.makeAbsolute(indexDataPath)
-    val newContent = Content.fromDirectory(absolutePath, fileIdTracker)
+    val hadoopConf = spark.sessionState.newHadoopConf()
+    val absolutePath = PathUtils.makeAbsolute(indexDataPath.toString)
```

Comment:
```suggestion
    val absolutePath = PathUtils.makeAbsolute(indexDataPath, hadoopConf)
```
as suggested here: https://github.com/microsoft/hyperspace/pull/327#discussion_r564142248

Reply:
👍 Good catch. I missed this one.